### PR TITLE
Release logo changes

### PR DESCRIPTION
**Pull request by tylerkelly13**
Related Ticket: WEBSITE-2819

Description of Changes:
* Release logo changes

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] `_data/nav.yml` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)
- [x] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

**Review by EkimChau — APPROVED** [review]

**Review by lnewson — APPROVED** [review]
Assuming this has already been checked by others since it was merged to `develop`, so I really haven't checked it very well.

**Review by ashfordneil — COMMENTED** [review]
Hey, looks like the vast majority of these new PNGs are "empty files" / "0 bytes long" according to GitHub. Don't really want to approve it until that's fixed (unless it's just a break on my end).

I've also gone through menu.html with a nicer diff viewer and left comments where all the actual changes are in that file (as noted in slack, for future PRs we should keep changes and auto-formatting separate).

File: _includes/template/menu.html [hunk 1]
```diff
@@ -1,258 +1,284 @@
...
-            <a class="link-menu" href="/" data-marketing="docs-header-back-home">
-                Return to Website
-            </a>
-        </nav>
+  {% include template/notification.html %}
+  <div class="nav-container">
+    <div class="nav-container__menu-container">
+      <div class="nav-container__homepage-link-container">
+        <a data-marketing="link" type="primary" href="/docs">
+          <img
```
[the remark this answers is not in the document]
Comment (tylerkelly13) [reply _includes/template/menu.html:194]: cool. intended. github is doing a weird thing, not sure why. The images do have size. See: https://www.staging.tiny.cloud/docs/

File: _includes/template/menu.html [hunk 2]
```diff
@@ -1,258 +1,284 @@
...
+            height="29px"
+          />
+        </a>
+      </div>
+
+      {% include template/search.html %}
+
+      <div class="nav-container__right-container">
+        <a
+          href="https://www.tiny.cloud/auth/signup/"
```
[the remark this answers is not in the document]
Comment (tylerkelly13) [reply _includes/template/menu.html:211]: Old link redirects to new link so I fixed it.

File: _includes/template/menu.html [hunk 3]
```diff
@@ -1,258 +1,284 @@
...
+  <div class="menu-mobile modern-menu-mobile">
+    <a
+      class="modern-menu-mobile__btn-open btn-open"
+      href="#"
+      title="Toggle menus"
+    ></a>
+    <nav class="menu-nav">
+      <a
+        class="link-menu header-get-tiny-button-mobile"
+        href="https://www.tiny.cloud/auth/signup/"
```
[the remark this answers is not in the document]
Comment (tylerkelly13) [reply _includes/template/menu.html:253]: Old link redirects to new link so I fixed it.

File: _includes/template/menu.html [hunk 4]
```diff
@@ -1,258 +1,284 @@
...
+        data-marketing="docs-header-get-started"
+        >Get Started</a
+      >
+
+      <a
+        href="https://www.tiny.cloud/my-account/dashboard/"
```
[the remark this answers is not in the document]
Comment (tylerkelly13) [reply _includes/template/menu.html:259]: Old link should redirect to new link so I fixed it.

File: _includes/template/menu.html [hunk 5]
```diff
@@ -1,258 +1,284 @@
...
+      <a
+        class="link-menu store-button-mobile"
+        href="https://apps.tiny.cloud/cart/"
+        data-marketing="tiny-news-cart-link"
+        >Cart</a
+      >
+
+      <a
+        class="link-menu store-button-mobile"
+        href="https://www.tiny.cloud/auth/signup/"
```
[the remark this answers is not in the document]
Comment (tylerkelly13) [reply _includes/template/menu.html:274]: Old link redirects to new link so I fixed it.

File: _includes/template/menu.html [hunk 6]
```diff
@@ -1,258 +1,284 @@
...
-                         alt="Tiny logo" height="32px">
-                </a>
-                <span>
```
[the remark this answers is not in the document]
Comment (tylerkelly13) [reply _includes/template/menu.html:199]: The old "logo" for tiny docs was the Tiny logo and the word docs after it. (feature not a bug :stuck_out_tongue:)

**Pull request merged**